docs/usage.md

```diff
@@ -197,6 +197,8 @@ The necessary fields in order are:
 
 You can optionally supply:
 
+- `formula` - A string representation of the model formula. It is used to build the design matrix.
+- `string` - An optional explicit literal contrast string (e.g., "treatmenthND6 - treatmentmCherry") that is passed directly to `limma::makeContrasts()` in `VARIANCEPARTITION_DREAM`. This field provides full control for complex designs. Requires `formula`.
```

Comment:
Could you expand this to illustrate what you might want to do here above and beyond the basic case?

Reply:
Agreed. A good example could be interaction terms, e.g `~ timepoint * treatment`. 

It could also be worth to mention the [`ExplorModelMatrix` shiny app](https://bioconductor.org/packages/release/bioc/vignettes/ExploreModelMatrix/inst/doc/ExploreModelMatrix.html) that helps understanding complex designs and building such contrast vectors.

Reply:
Thank you for your feedback. I added a better explanation for the use case. I also made a PR to shinyngs to change what's required from `validatefomcomponents`. I now also included an interaction case in the tests of `VARIANCEPARTITION_DREAM` https://github.com/nf-core/modules/pull/8231. Once those are merged I can update this PR.

Reply:
Once we get the update of [shinyngs ](https://github.com/bioconda/bioconda-recipes/pull/55567) the `VALIDATOR` module will be able to process the files with `make_contrasts_str` and we will make the dream test with different types of contrast.